src/python/turicreate/toolkits/sound_classifier/_mx_sound_classifier.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+
+import mxnet as mx
+import time
+import turicreate as _tc
+
+
+class SoundClassifierMXNETModel():
```

Comment:
Think bigger. There isn't anything here specific to sound classification. Maybe this can just be `MultiLayerPerceptionMXNetModel`

(Note: should be `MXNet` not `MXNET`, per the stylization I see on their web site: https://mxnet.incubator.apache.org (e.g. in the legal text at the bottom)